composer.json

```diff
@@ -11,26 +11,25 @@
 
     "incenteev/composer-parameter-handler": "^2.1",
 
-    "symfony/framework-bundle": "^3.3",
```

Comment:
Very good job. I've double checked them just in case, and all the dependencies are in their latest stable version. Congrats!!!